kura/distrib/src/main/deb/control-nvidia-jetson-nano/prerm

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
Would it be better if we have those files here in the specific target device folder like in here https://github.com/eclipse/kura/tree/develop/kura/distrib/src/main/resources/nvidia-jetson-nano?
What do you think?

Reply:
ok, but I need to do another round of tests to be sure there aren't broken paths to those files.